src/lib/challenge.ts

```diff
@@ -2,9 +2,14 @@
  License
  --------------
  Copyright © 2020 Mojaloop Foundation
```

Comment:
could you update the rest of the files with this compact License text?

Reply:
done